webdriver/elements/active.py

```diff
@@ -0,0 +1,257 @@
+# >    context's document element.
+# > 4. Let active web element be the JSON Serialization of active element.
+# > 5. Return success with data active web element.
+def test_sucess_document(session):
+    session.url = inline("""
+        <body>
+            <h1>Heading</h1>
+            <input />
+            <input />
+            <input style="opacity: 0;" />
+            <p>Another element</p>
+        </body>""")
+    result = session.transport.send("GET", "session/%s/element/active" % session.session_id)
+
+    assert_result_is_active_element(session, result)
+
+def test_sucess_input(session):
+    session.url = inline("""
+        <body>
+            <h1>Heading</h1>
+            <input autofocus />
+            <input style="opacity: 0;" />
+            <p>Another element</p>
+        </body>""")
+    result = session.transport.send("GET", "session/%s/element/active" % session.session_id)
+
+    assert_result_is_active_element(session, result)
+
+def test_sucess_input_non_interactable(session):
+    session.url = inline("""
+        <body>
+            <h1>Heading</h1>
+            <input />
+            <input style="opacity: 0;" autofocus />
+            <p>Another element</p>
+        </body>""")
+    result = session.transport.send("GET", "session/%s/element/active" % session.session_id)
+
+    assert_result_is_active_element(session, result)
+
+def test_success_explicit_focus(session):
+    session.url = inline("""
+        <body>
+            <h1>Heading</h1>
+            <input />
+            <iframe></iframe>
+        </body>""")
+
+    session.execute_script("document.body.getElementsByTagName('h1')[0].focus();")
+    result = session.transport.send("GET", "session/%s/element/active" % session.session_id)
+    assert_result_is_active_element(session, result)
+
+    session.execute_script("document.body.getElementsByTagName('input')[0].focus();")
+    result = session.transport.send("GET", "session/%s/element/active" % session.session_id)
+    assert_result_is_active_element(session, result)
+
+    session.execute_script("document.body.getElementsByTagName('iframe')[0].focus();")
+    result = session.transport.send("GET", "session/%s/element/active" % session.session_id)
+    assert_result_is_active_element(session, result)
+
+    session.execute_script("document.body.getElementsByTagName('iframe')[0].focus();")
+    session.execute_script("document.body.getElementsByTagName('iframe')[0].remove();")
+    result = session.transport.send("GET", "session/%s/element/active" % session.session_id)
+    assert_result_is_active_element(session, result)
+
+    session.execute_script("document.body.appendChild(document.createElement('textarea'));")
+    result = session.transport.send("GET", "session/%s/element/active" % session.session_id)
+    assert_result_is_active_element(session, result)
+
+def test_success_iframe_content(session):
+    session.url = inline("<body></body>")
```

Comment:
FWIW, `<body></body>`, and the use of the `<body>` tag in general, is unnecessary. (This is not a blocker.)